site/templates/application/footer/app-badges.jet

```diff
@@ -19,6 +22,33 @@
         {{end}}
       </div>
     </div>
+    {{yield separator()}} {* Test these - toggle configs and test resolutions *}
+  {{end}}
+  {{if config("app_link_apple_tv_enabled") == "true" || config("app_link_android_tv") != "" || config("app_link_fire_tv") != ""}}
+    <div class="tv-app-banner">
+      <h2 class="tv-app-badge-title">{{ i18n("tv_app_badge_title") }}</h2>
+      <div class="tv-app-badge-container">
+        {{if config("app_link_fire_tv") != ""}}
```

Comment:
These can all be extracted into a separate block to reduce repetition.  (The app badge ones could use the same extracted block).